ramlfications/errors.py

```diff
@@ -63,3 +63,7 @@ class MediaTypeError(Exception):
 
 class InvalidVersionError(Exception):
     pass
+
+
+class UnknownTypeError(Exception):
```

Comment:
This is a bit confusing, could we use `UnknownDataTypeError`?  Also I think this should inherit from `BaseRAMLError` like the other RAML-specific errors.